pad_server/src/gpio_actuator.c

```diff
@@ -0,0 +1,56 @@
+// BEWARE, this file is expluded from Linux compilation
+// because the of the missing nuttx header file
+
+#include "gpio_actuator.h"
+#include "stdio.h"
+#include <errno.h>
+#include <fcntl.h>
+#include <nuttx/ioexpander/gpio.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+
+int gpio_actuator_on(actuator_t *act) {
+    // TODO: change the device to be actuator specific
+    int fd = open("/dev/gpio12", O_RDWR);
```

Comment:
This is making me wonder if we should revise the actuator library later, cause it's gonna be cumbersome to open/close files on every call. We'll see if there's either a NuttX API way around this or if we need to make some changes to the way our code is structured. For now this is great